src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -456,3 +452,14 @@ def validate_namespace(namespace):
         return True
     else:
         return False
+
+
+
+def load_db_config():
+    '''
+    Load the database_global.json for multi asic
+    '''
+    if is_multi_asic():
+        swsscommon.SonicDBConfig.load_sonic_global_db_config()
+    else:
+        swsscommon.SonicDBConfig.initialize()
```

Comment:
Not a blocking issue. You may use `load_sonic_db_config` which is available in both swsscommon and swsssdk.